src/sass/_common/wallet_header.scss

```diff
@@ -0,0 +1,226 @@
+.wallet {
+    &__header {
+        box-sizing: border-box;
+        height: 48px;
+        position: fixed;
+        top: 0;
+        transition: opacity 0.3s;
+        width: 100%;
+        z-index: 99;
+        background-color: var(--general-main-1);
+        border-bottom: 1px solid var(--general-section-2);
+
+        &-demo-batch {
+            display: inline-flex;
+            font-size: $FONT_SIZE_XXS;
+            background-color: var(--status-info);
+            border-radius: 2px;
+            height: 14px;
+            padding-inline: 4px;
+            font-weight: bold;
+            color: var(--general-main-1);
+            align-items: center;
+            justify-content: center;
+            line-height: 1.5;
+        }
+        &-menu {
+            cursor: pointer;
+
+            &-left, &-right {
+                display: inline-flex;
+                height: 47px;
+
+                & > * {
+                    align-self: center;
```

Comment:
if all direct children have align-self: center;, would it be the same as if 
```
&-left, &-right {
                display: inline-flex;
                height: 47px;
                align-items: center;
}
```
?

Reply:
nope. what you've suggested would apply to the parent container instead of child items.